README.md

```diff
@@ -25,21 +25,24 @@ The diagram illustrates the usage of the connectivity model in k8s
 #### Add Dependency Repo
 ```bash
 # bootstrap a chart
-$ helm create example
+$ helm create example -n namespace
 
 # REQUIRED: add the helper library as the dependency to support usage
-$ helm repo add agent+helper https://ec-release.github.io/oci/agent+helper/<version. E.g. "0.1.0">
+$ helm repo add agent+helper https://ec-release.github.io/oci/agent+helper/<version. E.g. "0.1.0"> -n namespace
```

Comment:
pls replace "..ec-release.github.io.." to the following-

https://raw.githubusercontent.com/EC-Release/helmcharts/disty/agent+lber/<version>

and refactoring other links as well.

Reply:
README updated with new URL's